### PR TITLE
Disallow `given as <ident>` syntax

### PR DESCRIPTION
**Pull request by tgodzik**
Based on https://github.com/lampepfl/dotty/pull/10089

### CODE REVIEW

File: community-test/src/test/scala/CommunityDottySuite.scala [hunk 1]
```diff
@@ -171,15 +171,16 @@ class CommunityDottySuite extends FunSuite {
...
-    // Below syntax is no longer valid
-    // val result: List[Int] = fmap[F = List, A = Int, B = Int](List(1,2,3))(i => i + 1)
+    // Below syntax is no longer valid:
+    // - val result: List[Int] = fmap[F = List, A = Int, B = Int](List(1,2,3))(i => i + 1)
     "main/scala/NamedTypeArguments.scala",
-    // def (s: String).doesntStartWithAnyOfThese(c: Char*) = c.forall(char => !s.startsWith(char.toString))
-    "test/scala/dotty/dokka/DottyTestRunner.scala"
+    // - def (s: String).doesntStartWithAnyOfThese(c: Char*) = c.forall(char => !s.startsWith(char.toString))
+    "test/scala/dotty/dokka/DottyTestRunner.scala",
+    // - given as Context = ctx
```
Comment (kpbochenek) [inline community-test/src/test/scala/CommunityDottySuite.scala:182]: changes were merged to dotty, bumping community-dotty should fix it

**Review by kpbochenek — APPROVED** [review]
LGTM! 
(if bumping dotty-codebase only fix this one exclusion we could include that change in this PR, if it adds other problems then what you prefer, can be handled another time)

File: community-test/src/test/scala/CommunityDottySuite.scala (hunk 1 above)
Comment (tgodzik) [reply community-test/src/test/scala/CommunityDottySuite.scala:182]: It doesn't seem it was merged yet: https://github.com/lampepfl/dotty/pull/10081

**Comment by tgodzik** [comment]
Also checked and bumping dotty codebase causes the new given imports to break too. I will merge this one and follow up with a bump later on.

File: community-test/src/test/scala/CommunityDottySuite.scala (hunk 1 above)
Comment (kpbochenek) [reply community-test/src/test/scala/CommunityDottySuite.scala:182]: I referred to `lampepfl/dotty#10089` :) nvm, all good :+1:

**Pull request merged**